scripts/repl.js

```diff
@@ -1,5 +1,15 @@
 (function(babel, $, _, ace, window) {
 
+  /* Throw meaningful errors for getters of commonjs. */
+  ["module", "exports", "require"].forEach(function(commonVar){
+    Object.defineProperty(window, commonVar, { 
+      get: function () { 
+      throw new Error(commonVar + " is not supported in the browser,\
+        you need a commonjs environment such as node.js/io.js, browserify/webpack etc");
```

Comment:
Whoops, this line break causes the log to have a lot of extra spaces. @hemanth mind opening a new PR to remove this whitespace?